GDM.HW4.OOP.Classes.Task3/Program.cs

```diff
@@ -0,0 +1,371 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GDM.HW4.OOP.Classes.Task3
+{
+    class Program
+    {
+        private static Shop _shop;
+        private static MobileStore[] _mobStore;
+        private static Phone[][] _phone;
+        static void Main(string[] args)
+        {
+            SetShopName();
+            SetNumberOfStores();
+            _mobStore = new MobileStore[_shop.NumberOfStores];
+            _phone = new Phone[_shop.NumberOfStores][];
+            ConsoleOptionMenu();
+
+            Console.ReadLine();
+        }
+        public static void ConsoleOptionMenu ()
+        {
+            Console.WriteLine("- - - - - NEW COMMAND - - - - -");
+            Console.WriteLine("Please write index of command from list below. Commands: " +
+                "\n [0] - quit " +
```

Comment:
Imagine that you get commands from some method. Don't hardcode 28-32 lines. Do refactoring